modules/mirror-registry-introduction.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-introduction_{context}"]
+== Mirror registry for {product-title} introduction
+
+Mirror registry allows users to install a small-scale version of Red Hat Quay and its required components using the `mirror-registry` command line interface (CLI) tool. Mirror registry comes equipped with pre-configured local storage and a local database. It also includes pre-configured user credentials and access permissions with a single set of inputs and no additional configuration choices to get started.
```

Comment:
```suggestion
The mirror registry allows users to install a small-scale version of Red Hat Quay and its required components using the `mirror-registry` command line interface (CLI) tool. The mirror registry will be deployed fully automatically with pre-configured local storage and a local database. It also includes auto-generated user credentials and access permissions with a single set of inputs and no additional configuration choices to get started.
```

Reply:
Can't use 'will' (future tense) in OCP docs. Changing to 'The mirror registry is deployed. . . "